src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -46,16 +46,11 @@ public RazorPage()
             _writerScopes = new Stack<TextWriter>();
         }
 
-        public HttpContext Context
+        public HttpContext HttpContext
```

Comment:
I just noticed @rynowak's comment here - https://github.com/aspnet/Mvc/issues/3332#issuecomment-148739038 and we don't want to break compat here. Could you undo changes to this file?